<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>func</class>
 <widget class="QWidget" name="func">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>特色功能</string>
  </property>
  <widget class="QLabel" name="label_word">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>490</y>
     <width>1451</width>
     <height>541</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>得意黑</family>
     <pointsize>22</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>22</y>
     <width>168</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>得意黑</family>
     <pointsize>22</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>课表生成ics文件</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>71</y>
     <width>1478</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="button_excel">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>10000</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>得意黑</family>
          <pointsize>22</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>导入excel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>得意黑</family>
          <pointsize>22</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>路径</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_date">
        <property name="font">
         <font>
          <family>得意黑</family>
          <pointsize>16</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_excel_ics">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>得意黑</family>
          <pointsize>22</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>生成课表</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="maximumSize">
         <size>
          <width>191</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>得意黑</family>
          <pointsize>22</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>开学日期</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="date_excel">
        <property name="minimumSize">
         <size>
          <width>247</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>10000</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>得意黑</family>
          <pointsize>22</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2023</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="maximumSize">
         <size>
          <width>312</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>得意黑</family>
          <pointsize>22</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>提前时间（min）</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_starttime">
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>得意黑</family>
          <pointsize>22</pointsize>
          <italic>true</italic>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="maximumSize">
         <size>
          <width>230</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>得意黑</family>
          <pointsize>22</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>时间表名称</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_name">
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>得意黑</family>
          <pointsize>22</pointsize>
          <italic>true</italic>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>得意黑</family>
          <pointsize>22</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>日志生成ics文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_assignment_ics">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>得意黑</family>
          <pointsize>22</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>生成任务</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="maximumSize">
         <size>
          <width>191</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>得意黑</family>
          <pointsize>22</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>开始时间</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="data_time">
        <property name="font">
         <font>
          <family>得意黑</family>
          <pointsize>22</pointsize>
          <italic>true</italic>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="maximumSize">
         <size>
          <width>191</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>得意黑</family>
          <pointsize>22</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>结束时间</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="data_time_2">
        <property name="font">
         <font>
          <family>得意黑</family>
          <pointsize>22</pointsize>
          <italic>true</italic>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="maximumSize">
         <size>
          <width>312</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>得意黑</family>
          <pointsize>22</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>提前时间（min）</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_starttime2">
        <property name="maximumSize">
         <size>
          <width>112</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>得意黑</family>
          <pointsize>22</pointsize>
          <italic>true</italic>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="maximumSize">
         <size>
          <width>215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>得意黑</family>
          <pointsize>22</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>时间表名称</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_name_2">
        <property name="maximumSize">
         <size>
          <width>220</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>得意黑</family>
          <pointsize>22</pointsize>
          <italic>true</italic>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="maximumSize">
         <size>
          <width>164</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>得意黑</family>
          <pointsize>22</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>内容</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_title">
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>得意黑</family>
          <pointsize>22</pointsize>
          <italic>true</italic>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="maximumSize">
         <size>
          <width>165</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>得意黑</family>
          <pointsize>22</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>地点</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_place">
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>得意黑</family>
          <pointsize>22</pointsize>
          <italic>true</italic>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="maximumSize">
         <size>
          <width>170</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>得意黑</family>
          <pointsize>22</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>备注</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_remarks">
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>得意黑</family>
          <pointsize>22</pointsize>
          <italic>true</italic>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
